packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -166,6 +168,28 @@ class GestureRecognizerFactoryWithHandlers<T extends GestureRecognizer> extends
 /// ```
 /// {@end-tool}
 ///
+/// {@tool snippet}
+///
+/// This example of a [Container] wraps a [GestureDetector] widget.
+/// Since the [GestureDetector] does not have a child it takes on the size of
+/// its parent making the entire area of the surrounding [Container] clickable.
+/// When tapped the [Container] turns yellow by setting the `_color` field:
+///
+/// ```dart
+/// Container(
+///   color: _color,
+///   height: 200.0,
```

Comment:
This should probably also have a width if we specify a height.

Reply:
Done✔